src/handlers/countCurrentAnswers.js

```diff
@@ -0,0 +1,12 @@
+import { quizData } from '../data.js';
+
+function countCorrectAnswers() {
+  let count = 0;
+  for (const question of quizData.questions) {
+    if (question.correct === question.selected) {
+      count += 1;
+    }
+  }
+  return count;
```

Comment:
The side of my brain that wants to be "clever" wants to change it to something like this:

```javascript
return quizData.questions
    .filter(({correct, selected}) => correct === selected)
    .length;
```

or maybe even fancier with `reduce`:
```javascript
return quizData.questions.reduce(({correct, selected}, count) => count + (correct === selected), 0);
```

But looking at it now, what you have here is much better. So probably don't change it.